dist/monet.js

```diff
@@ -824,7 +835,7 @@
         return Return(a);
     };
     Free.liftF = function(functor) {
-        return Suspend(functor.map(Return));
+        return isFunction(functor) ? Suspend(compose(Return, functor)) : Suspend(functor.map(Return));
```

Comment:
I'm not completely sure about this one.  `liftF` lifts a functor into the Free monad.  Functions are functors, but I wonder if this should be a different method.  Could it lead to confusion?